src/app/helptext/storage/volumes/datasets/dataset-form.ts

```diff
@@ -77,6 +77,7 @@ dataset_form_deduplication_tooltip: T('Transparently reuse a single copy of dupl
  data to save space. Deduplication can improve storage capacity, but is RAM intensive. \
  Compressing data is generally recommended before using deduplication. Deduplicating data is \
  a one-way process. <b>Deduplicated data cannot be undeduplicated!</b>.'),
+dataset_form_deduplication_warning: T('Deduplication is ram intensive and premanently affects data'),
```

Comment:
```suggestion
dataset_form_deduplication_warning: T('This feature is memory-intensive and <b>permanently affects how the data is stored</b>. It is recommended to be very familiar with the benefits and drawbacks of deduplication before activating this feature.'),
```
Basic suggestion, best I can do this late on a Friday :P